rapid7_insightvm_cloud/icon_rapid7_insightvm_cloud/actions/get_scan/action.py

```diff
@@ -20,11 +20,11 @@ def run(self, params={}):
 
         try:
             response = self.connection.ivm_cloud_api.call_api("scan/" + scan_id, "GET")
-            return response
+            return response[1]
         except requests.RequestException as e:
             self.logger.error(e)
             raise PluginException(
                 cause=f"Failed to get a valid response from InsightVM with given asset ID '{scan_id}'.",
-                assistance=f"Response was {response.request.body}",
-                data=response.status_code,
+                assistance=f"Response was {response[1].request.body}",
+                data=response[1].status_code,
```

Comment:
```suggestion
                data=response[0],
```
This would just be the first element of the tuple based on the return of your API call function